.github/workflows/ci.yml

```diff
@@ -1,35 +1,57 @@
-name: tests
+name: ci
 
 on:
   pull_request:
-    types: ['opened', 'synchronize']
+    types: ["opened", "synchronize"]
     paths-ignore:
-      - 'README.md'
+      - "README.md"
   push:
     branches:
       - master
     paths-ignore:
-      - 'README.md'
+      - "README.md"
+
+env:
+  GO_VERSION: "1.16"
+
 jobs:
-  unit:
+  tests:
```

Comment:
Due to this rename, now required status on PR is changed, so someone needs to update the repository to reflect this or I can revert it back `s/tests/unit/`. Both are fine with me. /cc @clok @mavogel

Reply:
I have updated the rules on the PR checks.

Reply:
Awesome, so I'm gonna squash and merge this.